publication/2019-10-21_announcing_ngxs_3_6/article.md

```diff
@@ -7,6 +7,7 @@
 - 💥 New lifecycle hook `ngxsOnChanges`
 - 💦 Fixed Actions Stream Subscriptions Leak
 - 🚧 Improved type safety for children states
+- 🧤 Expose StateContextFactory, StateFactory
```

Comment:
I left this out because it is an internal api that is not semantically versioned and could be subject to change. We shouldn't include this in the announcement IMO.